docs/common/error-utilities.ts

```diff
@@ -202,6 +202,9 @@ const RENAMED_PAGES: Record<string, string> = {
   '/guides/push-notifications/': '/push-notifications/overview/',
   '/guides/using-fcm/': '/push-notifications/using-fcm/',
 
+  // Redirects based on Sentry (29/12/2022)
```

Comment:
```suggestion
  // Redirects based on Sentry
```

I think that we can skip the date, and aggregate all missing redirects coming from Sentry reports under this group.

Reply:
Sounds good. There are other comments like this too so I am going to merge them into one and resolve this.

Reply:
Done.

Reply:
Thank you! :)